gist/src/main/java/build/gist/presentation/GistSdk.kt

```diff
@@ -188,6 +189,14 @@ object GistSdk : Application.ActivityLifecycleCallbacks {
         }
     }
 
+    internal fun removePersistentMessage(message: Message) {
+        val gistProperties = GistMessageProperties.getGistProperties(message)
+        if (gistProperties.persistent) {
+            Log.i(GIST_TAG, "Persistent message dismissed, logging view")
```

Comment:
```suggestion
            Log.i(GIST_TAG, "Persistent message dismissed")
```

Does "logging view" add value to the log statement?

Reply:
Yeah, I think it helps to remind anyone debugging that the message should not be displayed again.